src/test/java/net/masterthought/cucumber/generators/TagsOverviewPageTest.java

```diff
@@ -66,13 +66,14 @@ public void prepareReport_setTagsToExcludeFromChart_ReturnsFilteredTags() {
 
         // then
         VelocityContext context = page.context;
-        assertThat(context.get("chart_categories")).isEqualTo("[\"@fast\"]");
-        assertThat(context.get("chart_data")).isEqualTo(asList(
-                "[100.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]"));
+        assertThat(context.get("chart_categories")).isEqualTo(new String[]{"@fast"});
```

Comment:
There is IntArrayAssert that allows these operations to work on arrays but is these are created through: `assertThat(int[] actual)`. Because `context.get()` always returns an object, the method binds to  `assertThat(T actual)` and we don't get that fancy matcher. .